src/endpoint/s3/s3_utils.js

```diff
@@ -322,6 +324,18 @@ function set_response_object_md(res, object_md) {
     }
 }
 
+/**
+ * @param {nb.S3Response} res 
+ * @param {Array<string>} [supported_storage_classes]
+ */
+function set_response_supported_storage_classes(res, supported_storage_classes) {
+    if (!supported_storage_classes) {
+        supported_storage_classes = [STORAGE_CLASS_STANDARD];
```

Comment:
For completeness, since this config is not specific to nsfs, I would check it here too -
```
        if (!config.DENY_UPLOAD_TO_STORAGE_CLASS_STANDARD) {
            storage_classes.push(s3_utils.STORAGE_CLASS_STANDARD);
        }
```